Xamarin.Android-Tests.sln

```diff
@@ -90,128 +92,195 @@ Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		Debug|Any CPU = Debug|Any CPU
 		Release|Any CPU = Release|Any CPU
+		Bundle|Any CPU = Bundle|Any CPU
```

Comment:
@jonpryor I really hate modifying the sln is VSForMac, it does all sorts or weird stuff.